source/content/guides/wordpress-composer/04-bedrock-composer.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Using Bedrock to Create Your WordPress Pantheon Site
+subtitle: Create a Composer-managed WordPress Site with Bedrock
+description: Learn more about Bedrock for Composer-managed WordPress sites.
+categories: [develop]
+tags: [wordpress]
+contributors: [whitneymeredith]
+layout: guide
+showtoc: true
+permalink: docs/guides/wordpress-composer/bedrock-composer
+anchorid: bedrock-composer
+---
+
+[Composer](https://getcomposer.org/) does not fully support WordPress, however, Bedrock is a central feature of Composer. [Bedrock](https://roots.io/bedrock/) is a custom WordPress project structure that helps simplify working on large projects with multiple plugins and custom code. 
```

Comment:
"Bedrock is a central feature of Composer" not sure what does this mean. Maybe of "Composer in the Wordpress ecosystem"?

Reply:
Agree with @kporras07. _Bedrock_ isn't a central feature of Composer as Composer exists far and wide outside of the WordPress-specific implementation in Bedrock. I would also argue that "Composer does not fully support WordPress" is not entirely accurate. I would rephrase this to:

```suggestion
WordPress does not natively support [Composer](https://getcomposer.org/), however, Bedrock is a WordPress-specific framework for using Composer on WordPress sites. [Bedrock](https://roots.io/bedrock/) is a custom WordPress project structure that helps simplify working on large projects with multiple plugins and custom code. 
```